lib/importers/licence_transaction/tagging_csv_validator.rb

```diff
@@ -0,0 +1,55 @@
+module Importers
+  module LicenceTransaction
+    class TaggingCsvValidator
+      attr_reader :licences_tagging
+
+      def initialize(licences_tagging)
+        @licences_tagging = licences_tagging
+      end
+
+      def valid?
+        tagging_validation_errors.empty?
+      end
+
+      def errors
+        tagging_validation_errors.each { |e| puts "#{e}\n\n" }
+
+        puts unrecognised_tags_instructions unless valid?
```

Comment:
You could put this line at the top of the method to avoid any extra processing.

i.e.

```ruby
  def errors
    return if valid?

    tagging_validation_errors.each { |e| puts "#{e}\n\n" }
    puts unrecognised_tags_instructions
  end

```

Reply:
Sure thing 👍 Done!